x-pack/plugins/monitoring/public/directives/all.js

```diff
@@ -27,3 +27,6 @@ import './kibana/summary';
 import './beats/overview';
 import './beats/listing';
 import './beats/beat';
+import './apm/instance';
+import './apm/instances';
+import './apm/overview';
```

Comment:
If we mount react components directly from the Angular view, we should not need any Angular directives. Over time with refactoring, we shouldn't need this file at all

Reply:
Great!